src/typed-method-types/workflows/triggers/mod.ts

```diff
@@ -17,11 +17,13 @@ type WorkflowInput = {
   value: any;
 };
 
+type WorkflowStringFormat = `${string}#/workflows/${string}`;
```

Comment:
wow had no idea you can do that - slick!

Reply:
Yeah, [Template Literals](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) are really cool! I actually saw a tweet about them validating url structures and remembered they existed